README.md

```diff
@@ -172,6 +172,7 @@ Travis uses a .travis.yml file in the root of your repository to learn about you
 | Key            | Description   |
 | :------------- |:--------------|
 |hiera_config|Sets the [`hiera_config`](http://rspec-puppet.com/documentation/configuration/#hiera_config) rspec-puppet parameter.|
+|hiera_execute_config|Sets the [`hiera_config`](http://rspec-puppet.com/documentation/configuration/#hiera_config) rspec-puppet parameter. It accepts command to find the path to your yaml file.|
```

Comment:
When I read it initially I had a hard time understanding what the difference to `hiera_config` is. Suggestions to improve the situation:

- [ ] use the word `ruby` instead of `execute` to be more specific about the content
- [ ] reword the second sentence to say `a ruby expression returning the path to your hiera.yaml file`` instead of `command ...` to be more specific and helpful.
- [ ] preserve the common prefix by using `hiera_config_ruby` instead of `hiera_ruby_config`. This will make this more searchable. That is folks searching for `hiera_config` will be able to find both
- [ ] specify how this and the regular `hiera_config` values interact. Is it an error if both values are set? Which one takes precedence?

Reply:
@DavidS Thanks for the valuable comment.Regarding the last comment.
`hiera_config` takes precedence if both values `hiera_config` and `hiera_config_ruby` are specified.
No errors are thrown if both values are specified.